src/dropdown/dropdown.component.ts

```diff
@@ -319,6 +324,28 @@ export class Dropdown implements OnInit, AfterContentInit, AfterViewInit, OnDest
 	@ViewChild("dropdownMenu", { static: true }) dropdownMenu;
 
 	@HostBinding("class.cds--dropdown__wrapper") hostClass = true;
+
+	@HostBinding("class.cds--list-box__wrapper") hostWrapperClass = true;
+	
+	/**
+	 * Experimental: enable fluid state
+	 */
+	@HostBinding("class.cds--list-box__wrapper--fluid") @Input() fluid = false;
+
+	/**
+	 * Style needed to properly display label in fluid layout.
+	 * React implementation is using a div that is a block-level element.
+	 */
+	@HostBinding("style.display") display = "block";
```

Comment:
We normally make a PR to @carbon/styles package and include `display` property with block. We can also use `:host` pseudo class, but I think we should avoid adding styles to maintain on our end. 

I'll make a PR in @carbon/styles and manually update the version here when it is merged. Will also look at other PRs to see if they contain display properties.

Reply:
Okay, it makes sense to have the style updated upstream. Thanks

Other PRs dont update the display property. Just this one.

Reply:
Sorry for the delay, waiting on one more approval on this. Hopefully it gets merged by end of this week!

Reply:
It's been merged! Will be released at the end of this week @1.71.0 🎉